bundles/framework/announcements/event/AnnouncementsChangedEvent.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @class Oskari.framework.announcements.event.AnnouncementsChangedEvent
+ *
+ * Used to notify getinfo plugin that the announcements have changed.
```

Comment:
I don't think this is used for notifying getinfo plugin of anything? If I read the code correctly it's used to only communicate changes in selections between the publisher tool and the plugin while publishing an embedded map. As such I would remove this event entirely.